posthog/async_migrations/test/test_0010_move_old_partitions.py

```diff
@@ -70,17 +70,18 @@ def test_completes_successfully(self):
         self.assertTrue(run_migration())
 
         # create table + 3 move operations
-        self.assertEqual(len(MIGRATION_DEFINITION.operations), 4)
+        self.assertEqual(len(MIGRATION_DEFINITION.operations), 5)
```

Comment:
This comment doesn't make a lot of sense with the subsequent line now since it's off by one. I'm curious (but don't know much about these migrations) -- what happened here?

Reply:
I have no idea and it applies to both versions so I'm assuming this has been broken for some time now.

Reply:
I looked at this for some time, but decided...since we'll probably nuke async migrations as they exist right now (please be temporal based) we can safely ignore this. It definitely won't impact production unless we decide to reuse async migration functionality.